pkg/scheduler/apis/config/v1beta1/zz_generated.conversion.go

```diff
@@ -213,8 +223,40 @@ func RegisterConversions(s *runtime.Scheme) error {
 	return nil
 }
 
+func autoConvert_v1beta1_Extender_To_config_Extender(in *v1beta1.Extender, out *config.Extender, s conversion.Scope) error {
+	out.URLPrefix = in.URLPrefix
+	out.FilterVerb = in.FilterVerb
+	out.PreemptVerb = in.PreemptVerb
+	out.PrioritizeVerb = in.PrioritizeVerb
+	out.Weight = in.Weight
+	out.BindVerb = in.BindVerb
+	out.EnableHTTPS = in.EnableHTTPS
+	out.TLSConfig = (*config.ExtenderTLSConfig)(unsafe.Pointer(in.TLSConfig))
+	// WARNING: in.HTTPTimeout requires manual conversion: inconvertible types (k8s.io/apimachinery/pkg/apis/meta/v1.Duration vs time.Duration)
```

Comment:
use meta/v1.Duration internally and avoid a manual conversion

Reply:
Will still need to do a manual conversion from `v1.Extender`.

Reply:
duplicate the type in `pkg/scheduler/apis/config/types.go` renaming the existing one as `LegacyExtender`.

Reply:
Actually, that might not work, please check... but you can do the "manual" conversion in the type that will eventually go away in v1 folder.

Reply:
If we duplicated `Extender` as `LegacyExtender`, we would still have to maintain a translation between them as we would use only one internally. I think it's better to do the manual conversion from `v1.LegacyExtender`.

Reply:
correct, only one internally and do manual conversion from the legacy one.